src/org/loklak/LoklakServer.java

```diff
@@ -167,49 +173,200 @@ public static void main(String[] args) throws Exception {
             if (ss != null) {try {ss.close();} catch (IOException e) {}}
         }
         
+        // check for redirect to https
+        String httpsMode = config.get("https.mode");
+        boolean redirect = httpsMode.equals("redirect");
+        boolean useHttps = redirect || httpsMode.equals("on");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/sudheesh1995/loklak_server/file/2550015915/issues/source?bid=3288751&fileBranchId=3288809#l179)